app/src/main/java/com/sun_asterisk/moviedb_50/data/source/remote/OnFetchDataJsonListener.kt

```diff
@@ -0,0 +1,8 @@
+package com.sun_asterisk.moviedb_50.data.source.remote
+
+import java.lang.Exception
+
+interface OnFetchDataJsonListener<T> {
+    fun onSuccess(data: MutableList<T>?)
```

Comment:
dao dữ liệu trả về luôn là 1 MutableList<> được 
trong trường hợp mình muốn chỉ lấy 1 cái thôi thì sao 
```suggestion
    fun onSuccess(data: T?)
```

Reply:
muốn dữ liệu trả về 1 list, thì hay tạo một callback 
``` Kotlin
OnFetchDataJsonListener<MutableList<SomeThing>>
```